homeassistant/components/totalconnect/alarm_control_panel.py

```diff
@@ -32,10 +32,12 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
 
     client = hass.data[TOTALCONNECT_DOMAIN].client
 
-    for location in client.locations:
-        location_id = location.get("LocationID")
-        name = location.get("LocationName")
-        alarms.append(TotalConnectAlarm(name, location_id, client))
+    for location_id in client.locations:
+        alarms.append(
+            TotalConnectAlarm(
+                client.locations[location_id].location_name, location_id, client
```

Comment:
Please create a local variable for the location_name beforehand and use it here to make it more readable.

Reply:
concur